configs/bs/cluster/disks.yaml

```diff
@@ -0,0 +1,18 @@
+global:
+  format_percent: 90
+
```

Comment:
I think maybe we should provide the global`hosts` section to let user specify the all host which will to formatted, otherwise user needs to setting many `hosts_exclude` or `hosts_only` if there are many hosts.  if using `hosts`, user can focus on a part hosts quicky, and then using `hosts_exclude` and `hosts_only` to achieve its aim.

```
global:
  format_percent: 95
  hosts:
    - curve1
    - curve2
    - curve3

disks:
  ...
```

BTW,  how do you think about the below configure design? I think:
* It might be more easier to understand.
* The [level design](https://github.com/opencurve/curveadm/wiki/topology#%E5%B1%82%E7%BA%A7) `SAME AS` our other configures, like topology.
* The logic of parse disks configure file is more simpler.
```
global:
  format_percent: 95
  hosts:
    - curve-1
    - curve-2
    - curve-3
   
  disks:
    - device: /dev/sdb1
       mount: /data/chunkserver0
    - device: /dev/sdc1
       mount: /data/chunkserver1
       format_percent: 90
    - device: /dev/sdd1
       mount: /data/chunkserver2
       hosts:
         - curve-1
         - curve-2
    - device: /dev/sde1
       mount: /data/chunkserver3
       hosts:
         - curve1
```

Reply:
It may be redundant to give host list in `disks.yaml` as we already have host information committed by `hosts.yaml`. Why should we put all the hosts we get in hand or not into `hosts.yaml` if we do not want to use some of the hosts.

Reply:
Done.